integration_test/buckets.tf.template

```diff
@@ -6,6 +6,25 @@ resource "aws_s3_bucket" "internal" {
     Application = "ORCA"
   }
 }
+# todo: Either commit to us-west-2 and remove user's ability to change it, or make 797873946194 variable with instructions.
+data "aws_iam_policy_document" "internal-policy-json" {
+ statement {
+   principals {
+     type        = "AWS"
+     identifiers = ["arn:aws:iam::797873946194:root"]
+    }
+    actions = [
+                "s3:PutObject",
+    ]
+    resources = [
+                "arn:aws:s3:::PREFIX-internal/PREFIX-lb-gql-a-logs/AWSLogs/236859827343/*"
```

Comment:
Is everything expexted to be in the numbered tag above `236859827343`? Should it be wildcarded at AWSLogs instead?

Reply:
This matches our [real policy](https://github.com/nasa/cumulus-orca/blob/b92ac6dc77830658d687494bc3438a814fbf4e6a/website/docs/developer/deployment-guide/creating-orca-archive-bucket.md?plain=1#L309). For this particular case, the policy does not need to be more permissive.